xml/admin_monitoring_alerting.xml

```diff
@@ -1123,11 +1119,11 @@ monitoring:
         <term><option>MGR_MODULE_DEPENDENCY</option></term>
         <listitem>
           <para>An enabled manager module is failing its dependency check.
-            This health check should come with an explanatory message from
+            This health check should come with a message from
             the module about the problem. For example, a module might report
-            that a required package is not installed: install the required
-            package and restart your manager daemons. This health check is only
-            applied to enabled modules. If a module is not enabled, you
+            that a required package is not installed, the message will read: "Install the required
```

Comment:
Comma splice.
https://en.wikipedia.org/wiki/Comma_splice

Suggest "in which case, the message will read"